<launch>
  <node name="frontier_extraction" pkg="frontier_exploration" type="frontier_extraction">

    <param name="info_gain_range" value="4.0" />
    <param name="LAMBDA" value="0.02" />

    <remap from="octomap_in" to="uav1/octomap_server/octomap_local_full" />
    <remap from="odom_in" to="uav1/estimation_manager/odom_main" />

  </node>
</launch>